src/components/Section.js

```diff
@@ -13,12 +13,12 @@ export default class Section {
   }
 
   //render all card elements
-  renderItems() {
-    this._items.forEach((cardData) => {
-      const cardItem = this._renderer(cardData);
-      this.appendItem(cardItem);
-    });
-  }
+  // renderItems() {
```

Comment:
You need to use `renderItems` for rending the array. You should just change it so that it could accept the array into the call


```js
  renderItems(items) {
    items.forEach((cardData) => {
      const cardItem = this._renderer(cardData);
      this.appendItem(cardItem);
    });
  }

```